project/plugins.sbt

```diff
@@ -10,5 +10,5 @@ addSbtPlugin("org.wartremover" % "sbt-wartremover" % "2.4.15")
 
 // Warning: These must be synced with
 // https://github.com/cognitedata/cdp-spark-datasource/blob/master/project/protoc.sbt
-addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.0") // See warning above
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.4") // See warning above
```

Comment:
@wjoel good warning! Just stopped me from auto-merging :sweat_smile:

Reply:
@hakontro hehe, yeah, but that might actually not be an issue now that we're just using the plain Java classes. We haven't updated the Spark data source yet though, so I guess it's still valid.